### PR TITLE
Make it work for 25

### PR DESCRIPTION
**Pull request by ziopio**
One thing it should be tested is if this PR is still able to work with 23 and 24.

### CODE REVIEW

File: src/grisp_cryptoauth_api_server.erl [hunk 1]
```diff
@@ -26,7 +26,7 @@ handle_call({Fun, Args}, _From, {Context, OldTRef}) ->
     %% on every API call reset a timer to put the device
     %% to sleep after SLEEP_TIME_SEC seconds to save energy
     timer:cancel(OldTRef), %% this doesn't throw on bad args
-    {ok, NewTRef} = timer:apply_after(?SLEEP_TIME_SEC * 1000, grisp_cryptoauth, sleep, Context),
+    {ok, NewTRef} = timer:apply_after(?SLEEP_TIME_SEC * 1000, grisp_cryptoauth, sleep, [Context]),
```
Comment (ziopio) [inline src/grisp_cryptoauth_api_server.erl:29]: TODO: test this does not break 23 or 24 prebuilts

**Comment by ziopio** [comment]
@GalaxyGorilla I have uploaded a prebuilt for 25 so you should be able to just deploy.

**Comment by GalaxyGorilla** [comment]
Hmm I think we should have the patch in one place, so I'd rather remove it from `grisp` and out everything here.

@sylane do you remember why patches related to cryptoauthlib were also put into `grisp`, e.g. https://github.com/grisp/grisp/blob/master/grisp/default/24/build/patches/00100-rtems.patch#L1862 ?

**Comment by GalaxyGorilla** [comment]
@ziopio I just talked with @sylane about this and the `grisp_cryptoauth` patches in `grisp` are just there by accident. Could you remove these patches (for OTP 24 and 25) in `grisp` and put everything here?

**Comment by ziopio** [comment]
@GalaxyGorilla ok, I proceed!

File: src/grisp_cryptoauth_api_server.erl (hunk 1 above)
Comment (ziopio) [reply src/grisp_cryptoauth_api_server.erl:29]: Tested, gave no problems.

**Pull request merged**